### PR TITLE
update doc

### PR DESCRIPTION
**Pull request by jiqing-feng**
Update CPU doc

### CODE REVIEW

File: docs/source/en/perf_train_cpu.md [hunk 1]
```diff
@@ -18,11 +18,11 @@ rendered properly in your Markdown viewer.
 This guide focuses on training large models efficiently on CPU.
 
 ## Mixed precision with IPEX
-Mixed precision uses single (fp32) and half-precision (bf16/fp16) data types in a model to accelerate training or inference while still preserving much of the single-precision accuracy. Modern CPUs such as 3rd and 4th Gen Intel® Xeon® Scalable processors natively support bf16, so you should get more performance out of the box by enabling mixed precision training with bf16.
+Mixed precision uses single (fp32) and half-precision (bf16/fp16) data types in a model to accelerate training or inference while still preserving much of the single-precision accuracy. Modern CPUs such as 3rd and 4th and 5th Gen Intel® Xeon® Scalable processors natively support bf16, and 6th Gen Intel® Xeon® Scalable processors natively support bf16 and fp16. You should get more performance out of the box by enabling mixed precision training with bf16 or fp16.
```
Comment (stevhliu) [inline docs/source/en/perf_train_cpu.md:21]:
```suggestion
Mixed precision uses single (fp32) and half-precision (bf16/fp16) data types in a model to accelerate training or inference while still preserving much of the single-precision accuracy. Modern CPUs such as 3rd, 4th, and 5th Gen Intel® Xeon® Scalable processors natively support bf16. 6th Gen Intel® Xeon® Scalable processors natively support bf16 and fp16. You should get more performance out of the box by enabling mixed precision training with bf16 or fp16.
```

File: docs/source/en/perf_train_cpu_many.md [hunk 2]
```diff
@@ -30,26 +30,21 @@ Check more detailed information for [oneccl_bind_pt](https://github.com/intel/to
...
 Please run `pip list | grep torch` to get your `pytorch_version`.
 ```bash
 pip install oneccl_bind_pt=={pytorch_version} -f https://developer.intel.com/ipex-whl-stable-cpu
 ```
-where `{pytorch_version}` should be your PyTorch version, for instance 2.1.0.
+where `{pytorch_version}` should be your PyTorch version, for instance 2.4.0.
 Check more approaches for [oneccl_bind_pt installation](https://github.com/intel/torch-ccl).
 Versions of oneCCL and PyTorch must match.
 
-<Tip warning={true}>
```
Comment (stevhliu) [inline docs/source/en/perf_train_cpu_many.md:49]: Need to also delete the closing `</Tip>` tag

**Review by stevhliu — COMMENTED** [review]
Thanks for updating! Feel free to mark as ready whenever you're ready :)

**Comment by jiqing-feng** [comment]
Hi @stevhliu . Thanks for your review. I have fixed all the comments, now it's ready to merge :)

**Review by stevhliu — APPROVED** [review]
Thanks! 🤗

**Pull request merged**